Sparkle/SUUpdateAlert.h

```diff
@@ -11,8 +11,9 @@
 
 #import "SUWindowController.h"
 #import "SUVersionDisplayProtocol.h"
+#import <WebKit/WebKit.h>
 
-@protocol SUUpdateAlertDelegate;
+@protocol SUUpdateAlertDelegate, WebPolicyDelegate, WebFrameLoadDelegate;
```

Comment:
What's the point of the forward declarations here? They're already present from WebKit.h.